metaserver/src/main/resources/logback.groovy

```diff
@@ -13,3 +13,5 @@ appender("LSP", scala.meta.languageserver.LSPLogger) {
 }
 
 root(DEBUG, ["LSP", "STDOUT"])
+logger("langserver.core.MessageWriter", INFO, ["LSP", "STDOUT"])
+logger("langserver.core.MessageReader", INFO, ["LSP", "STDOUT"])
```

Comment:
@olafurpg this should be easier to tweak. I've opted for raising the level to `INFO`, since the noisy RPC messages have a DEBUG level.